expression/builtin_arithmetic.go

```diff
@@ -453,6 +456,77 @@ func (s *builtinArithmeticMultiplyIntSig) evalInt(row []types.Datum) (val int64,
 	return result, false, nil
 }
 
+type arithmeticDivideFunctionClass struct {
+	baseFunctionClass
+}
+
+func (c *arithmeticDivideFunctionClass) getFunction(args []Expression, ctx context.Context) (builtinFunc, error) {
+	if err := c.verifyArgs(args); err != nil {
+		return nil, errors.Trace(err)
+	}
+	tpA, tpB := args[0].GetType(), args[1].GetType()
+	tcA, tcB := numericContextResultType(tpA), numericContextResultType(tpB)
+	if tcA == types.ClassReal || tcB == types.ClassReal {
+		bf, err := newBaseBuiltinFuncWithTp(args, ctx, tpReal, tpReal, tpReal)
+		if err != nil {
+			return nil, errors.Trace(err)
+		}
+		setFlenDecimal4RealOrDecimal(bf.tp, tpA, tpB, true)
```

Comment:
in TiDB:
```sql
TiDB > select 1.99 * 2.991;
Field   1:  `1.99 * 2.991`
Catalog:    `def`
Database:   ``
Table:      ``
Org_table:  ``
Type:       NEWDECIMAL
Collation:  binary (63)
Length:     8
Max_length: 7
Decimals:   3
Flags:      BINARY NUM


+--------------+
| 1.99 * 2.991 |
+--------------+
|      5.95209 |
+--------------+
1 row in set (0.00 sec)
```

in MySQL:
```sql
MySQL > select 1.99 * 2.991;
Field   1:  `1.99 * 2.991`
Catalog:    `def`
Database:   ``
Table:      ``
Org_table:  ``
Type:       NEWDECIMAL
Collation:  binary (63)
Length:     9
Max_length: 7
Decimals:   5
Flags:      NOT_NULL BINARY NUM


+--------------+
| 1.99 * 2.991 |
+--------------+
|      5.95209 |
+--------------+
1 row in set (0.00 sec)
```

`Flen` and `Decimal` can not be set using `setFlenDecimal4RealOrDecimal` for builtin function div and mul